src/resources/js/router.js

```diff
@@ -56,6 +57,11 @@ const router = new VueRouter({
       name: 'Redéfinir mot de passe',
       component: ResetPassword,
     },
+    {
+      path: '/presentation/modifier/:idPresentation',
```

Comment:
a good syntax for routes is `/<resource>/(<id>/)<action>` which in this case should be like `/presentation/:idPresentation/modifier`

Reply:
Changed in 33c1d34